R/rlistings.R

```diff
@@ -89,28 +121,31 @@ as_keycol <- function(vec) {
 }
 
 
-#' @export
+
 #' @param vec any. A column vector from a `listing_df`
-#'
 #' @rdname listings
+#' @export
 is_keycol <- function(vec) {
   inherits(vec, "listing_keycol")
 }
 
 
 
-#' @export
+
 #' @param df listing_df. The listing
 #' @rdname listings
+#' @export
 get_keycols <- function(df) {
   names(which(sapply(df, is_keycol)))
 }
 
-#' @export
+
 #' @inheritParams formatters::matrix_form
+#' @seealso [formatters::matrix_form()]
```

Comment:
Good! You may also add a short description specifying that this is partially inherited from formatters' function